### DISCUSSION TITLE
openshift v 4.5 installation on baremetal.

### DISCUSSION THREAD

**Discussion by saikiran982** [okd-project/okd#832] [Q&A]
I am installing open shift v 4.5 on bare metal as user provisioned interface with static network configurations.
When i am booting the red hat core os with ignition file and network configuration. Its throwing error as 
"dracut initqueue timeout:

can someone provide the solution for this

**Comment by Mateuy** [comment]
Hello, can you provide dracut stack error and ignition configurations ?
